pkg/extensions/main.go

```diff
@@ -1,10 +1,15 @@
 package extensions
 
 import (
+	// Need to remove "github.com/robfig/cron" with v3
```

Comment:
Is there plans for using the cron dependency outside of enterprise? Or do you mean something else?

Reply:
The `ldapsync` uses cron v1 and new stuff using v3, this comment basically means that `ldapsync` should switch to newer version of the cron after we have the time, was planning to create an issue when this pr lands

Reply:
okay. Suggestion for comment: // Move ldapsync from cron to cron.v3 and remove the cron (v1) dependency.

Reply:
Done!